code/_onclick/hud/alert.dm

```diff
@@ -356,11 +356,11 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 // CLOCKCULT
 /obj/screen/alert/clockwork
 	alerttooltipstyle = "clockcult"
-
+/*
```

Comment:
It'll be deleted once I'm done with the rework, but I'd like to keep it around for now for reference.